test-integration/workflow-samples/angular.json

```diff
@@ -94,8 +94,8 @@
       "entity": "couchbase"
     },
     {
-      "name": "ms-ng-eureka-oauth2-mongodb-caffeine",
-      "jdl-samples": "ms-ng-eureka-oauth2-mongodb-caffeine",
+      "name": "ms-ngx-consul-oauth2-mongodb-caffeine",
+      "jdl-samples": "ms-ngx-consul-oauth2-mongodb-caffeine",
```

Comment:
I think we should just use `ng` rather than `ngx`. I believe we removed most instances of `ngx` a couple of months ago.